### PR TITLE
No persistence

### PR DESCRIPTION
**Pull request by melroy89**
Explain how to configure no persistence (eg. when using Valkey as a cache, without the need for storage).

Related discussion: https://github.com/orgs/valkey-io/discussions/1134

### CODE REVIEW

**Comment by amirreza8002** [comment]
hi
maybe the command line options should be mentioned too.
for my case (docker) command line seems much easier

**Comment by melroy89** [comment]
> hi maybe the command line options should be mentioned too. for my case (docker) command line seems much easier

Added!

File: topics/persistence.md [hunk 1]
```diff
@@ -68,6 +68,17 @@ dataset to disk every 60 seconds if at least 1000 keys changed:
 
 This strategy is known as _snapshotting_.
 
+## No persistence
+
+If you prefer **not** to have persistence (eg. when using a Valkey instance solely as a cache) that is also possibility.
```
Comment (hpatro) [inline topics/persistence.md:73]: *a possibility

File: topics/persistence.md [hunk 2]
```diff
@@ -68,6 +68,17 @@ dataset to disk every 60 seconds if at least 1000 keys changed:
 
 This strategy is known as _snapshotting_.
 
+## No persistence
+
+If you prefer **not** to have persistence (eg. when using a Valkey instance solely as a cache) that is also a possibility.
+
+First comment out or remove all existing `save` lines that are present in the current configuration file.   
+Then add the following configuration instead:
```
Comment (zuiderkwast) [inline topics/persistence.md:76]: This sort of assumes there's a lot of junk in the user's config file. I'd prefer to focus first on the correct config and only afterwards mention what to do with extra junk in the files, if any. How about this?

```suggestion
RDB snapshotting is enabled by default. To disable it, set the `save` configuration parameter to the empty string `""` and remove any excess `save` lines that are present in the current configuration file.
```

**Review by zuiderkwast — COMMENTED** [review]
Very nice, thanks! I added a small suggestion.

File: topics/persistence.md [hunk 3]
```diff
@@ -68,6 +68,17 @@ dataset to disk every 60 seconds if at least 1000 keys changed:
 
 This strategy is known as _snapshotting_.
 
+## No persistence
+
+If you prefer **not** to have persistence (eg. when using a Valkey instance solely as a cache) that is also a possibility.
```
Comment (zuiderkwast) [inline topics/persistence.md:73]: The spellchecker complains about "eg." I think it wants it spelled "e.g.", but I recall now that we've avoided abbreviatinos like this in the docs in the past. Maybe "for example" is clearer?

```suggestion
If you prefer **not** to have persistence (for example when using a Valkey instance solely as a cache) that is also a possibility.
```

Comment (melroy89) [reply topics/persistence.md:73]: Agreed my bad. I also also reluctant.

**Review by zuiderkwast — APPROVED** [review]
Aweseome. Thanks!

If you want help more with docs, there are some open issues in this repo. :)

**Pull request merged**